core/templates/dev/head/pages/topic-editor-page/editor-tab/stories-list.directive.html

```diff
@@ -26,7 +26,7 @@
     </td>
     <td>
       <a ng-click="openStoryEditor(story.id)" class="list-summary">
-        <span> <[story.story_publication_status]> </span>
+        <span class="protractor-test-story-publication-status"> <[story.story_publication_status ? "Yes" : "No"]> </span>
```

Comment:
Why is this using snake_case? Shouldn't `story` be a domain object with lowerCamelCase properties (i.e. storyPublicationStatus)?

If `story` isn't a domain object already, please make it so.

Reply:
Done, created a story summary domain object